src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/RequiredReadInputArgumentCollection.java

```diff
@@ -13,6 +14,7 @@
  */
 public final class RequiredReadInputArgumentCollection extends ReadInputArgumentCollection {
     private static final long serialVersionUID = 1L;
+    @WorkflowInput(optionalCompanions = StandardArgumentDefinitions.INPUT_INDEX_COMPANION)
```

Comment:
Leave some space between the `serialVersionUID` and this annotation for readability. Also, you should pick a canonical ordering for `@Argument` and `WorkflowInput` and stick to it consistently. Above you had `@Argument` coming first, but here you have `WorkflowInput` first.

Reply:
Done. I did try to use the same ordering everywhere but I guess I missed this one - I don't see any others.